components/Header.tsx

```diff
@@ -54,12 +54,28 @@ export default function Header({ address, isConnected }: Props) {
           </button>
         </>
       ) : (
-        <button
-          className="primary-button"
-          onClick={() => connect({ connector: connectors[0] })}
-        >
-          Log in
-        </button>
+        <>
```

Comment:
do we need a selection here, as the dropdown refactor prompts directly
should we do `Login` default --> sequence without options for smoother UX @gglucass

Reply:
I believe this was done like this so we have _something_ in place before @Riglanto went on a mini 2-day holiday.

The high level design should become like here: https://www.figma.com/file/396p2PzjnIXSbHmuBuj7oK/Website?type=design&node-id=3531%3A6970&t=Wug69Uvs9EqQOG9u-1